Description: Fix torbrowser-launcher profile
 firejail (0.9.58-1parrot5) testing; urgency=medium
 .
   * Fix torbrowser-launcher sandbox proile.
   * Fix keepassxc sandbox profile.
Author: Lorenzo "Palinuro" Faletra <palinuro@parrotsec.org>
Last-Update: 2019-02-14

--- firejail-0.9.58.orig/etc/torbrowser-launcher.profile
+++ firejail-0.9.58/etc/torbrowser-launcher.profile
@@ -35,13 +35,13 @@ netfilter
 nodbus
 nodvd
 nogroups
-nonewprivs
+#nonewprivs
 noroot
 notv
 nou2f
 novideo
-protocol unix,inet,inet6
-seccomp.drop @clock,@cpu-emulation,@debug,@module,@obsolete,@raw-io,@reboot,@resources,@swap,acct,add_key,bpf,fanotify_init,io_cancel,io_destroy,io_getevents,io_setup,io_submit,ioprio_set,kcmp,keyctl,mount,name_to_handle_at,nfsservctl,ni_syscall,open_by_handle_at,personality,pivot_root,process_vm_readv,ptrace,remap_file_pages,request_key,setdomainname,sethostname,syslog,umount,umount2,userfaultfd,vhangup,vmsplice
+#protocol unix,inet,inet6
+#seccomp.drop @clock,@cpu-emulation,@debug,@module,@obsolete,@raw-io,@reboot,@resources,@swap,acct,add_key,bpf,fanotify_init,io_cancel,io_destroy,io_getevents,io_setup,io_submit,ioprio_set,kcmp,keyctl,mount,name_to_handle_at,nfsservctl,ni_syscall,open_by_handle_at,personality,pivot_root,process_vm_readv,ptrace,remap_file_pages,request_key,setdomainname,sethostname,syslog,umount,umount2,userfaultfd,vhangup,vmsplice
 shell none
 # tracelog may cause issues, see github issue #1930
 #tracelog
